rust/operator-binary/src/controller.rs

```diff
@@ -652,22 +648,17 @@ fn build_rolegroup_statefulset(
     // we need to mount ldap bind credentials from the secret as env vars
     if let Some(auth) = authentication_config {
         match auth {
-            TrinoAuthenticationConfig::MultiUser { .. } => {}
+            TrinoAuthenticationConfig::MultiUser { .. } => {
+                cb_prepare.add_volume_mount("users", USER_PASSWORD_DATA_DIR_NAME);
+                cb_trino.add_volume_mount("users", USER_PASSWORD_DATA_DIR_NAME);
+                pod_builder.add_volume(
+                    VolumeBuilder::new("users")
+                        .with_empty_dir(Some(""), None)
```

Comment:
I would prefer `.with_empty_dir(None, None)` (as we don't want to set any medium) for the whole file (5 matches)

Reply:
done